src/meili/routing.cc

```diff
@@ -543,6 +543,11 @@ find_shortest_path(baldr::GraphReader& reader,
             turn_cost += turn_cost_table[0];
           }
 
+          auto opp_edgeid = reader.GetOpposingEdgeId(origin_edge.id);
```

Comment:
We probably need a comment up above this somewhere describing that we're checking for the forward edge and the reverse edge (under the case of a u-turn, for example) .